### PR TITLE
CentOS Stream 9 support

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,4 +20,9 @@ gem 'puppet-blacksmith', '>= 6.0.0', {"groups"=>["development"]}
 gem 'voxpupuli-acceptance', '~> 1.0', {"groups"=>["system_tests"]}
 gem 'puppetlabs_spec_helper', {"groups"=>["system_tests"]}
 
+# EL9 support
+gem 'beaker', github: 'ekohl/beaker', branch: 'EL9-support'
+gem 'beaker-docker', github: 'ekohl/beaker-docker', branch: 'el9'
+gem 'beaker-hostgenerator', github: 'ekohl/beaker-hostgenerator', branch: 'centos9'
```
Comment (evgeni) [inline Gemfile:26]: Love the consistency in the branch names 🤣

Comment (ekohl) [reply Gemfile:26]: While writing this I **knew** you'd notice :P

**Comment by ekohl** [comment]
I noticed Puppet has released EL9 packages so that unblocked me. I found some other work that should be done in beaker (like don't install curl, it conflicts with curl-minimal). However, it's now at the point where it fails on real EL9 issues rather than infra. I'll do my best to continue to get beaker ready.

**Comment by ekohl** [comment]
Looks like there's no TFTP server nor client in CentOS 9.

```console
# dnf search tftp
CentOS Stream 9 - BaseOS                                                                                                                              7.7 MB/s | 5.3 MB     00:00    
CentOS Stream 9 - AppStream                                                                                                                            19 MB/s |  11 MB     00:00    
CentOS Stream 9 - Extras packages                                                                                                                     434  B/s | 257  B     00:00    
============================================================================ Name & Summary Matched: tftp ============================================================================
syslinux-tftpboot.noarch : SYSLINUX modules in /tftpboot, available for network booting
```

**Comment by lzap** [comment]
Let's wait what base OS will say about it, IMHO we will need to replace it with dnsmasq. That should be sufficient, probably not great DHCP/DNS server but should do for TFTP: https://bugzilla.redhat.com/show_bug.cgi?id=2029229

**Comment by ekohl** [comment]
Per https://bugzilla.redhat.com/show_bug.cgi?id=2029229#c13 it looks like it'll come back in EL9.

**Comment by ekohl** [comment]
Rebasing to see if it's already present CentOS Stream 9. I didn't check the real sources.

**Review by ehelms — APPROVED** [review]

**Pull request merged**